go.mod

```diff
@@ -1,100 +1,33 @@
-	github.com/go-openapi/swag v0.19.12
-	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/google/go-cmp v0.5.4 // indirect
-	github.com/google/uuid v1.1.2
-	github.com/imdario/mergo v0.3.10 // indirect
-	github.com/infinispan/infinispan-operator v0.0.0-20210621093106-4662500f4ae1
-	github.com/integr8ly/grafana-operator/v3 v3.10.0
-	github.com/keycloak/keycloak-operator v0.0.0-20200917060808-9858b19ca8bf
+	github.com/RHsyseng/operator-utils v0.0.0-00010101000000-000000000000
+	github.com/go-logr/logr v0.4.0
+	github.com/google/uuid v1.3.0
 	github.com/kiegroup/kogito-operator/api v0.0.0-00010101000000-000000000000
-	github.com/machinebox/graphql v0.2.2
-	github.com/matryer/is v1.4.0 // indirect
-	github.com/mongodb/mongodb-kubernetes-operator v0.3.0
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.2
-	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/openshift/client-go v3.9.0+incompatible
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
-	github.com/operator-framework/operator-marketplace v0.0.0-20190919183128-4ef67b2f50e9
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.46.0
-	github.com/prometheus/client_golang v1.9.0 // indirect
-	github.com/sirupsen/logrus v1.7.0 // indirect
-	github.com/spf13/cobra v1.1.1
-	github.com/spf13/pflag v1.0.5
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
+	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
+	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.50.0
+	github.com/spf13/cobra v1.1.3
 	github.com/stretchr/testify v1.7.0
-	go.uber.org/atomic v1.7.0 // indirect
-	go.uber.org/zap v1.15.0
-	golang.org/x/net v0.0.0-20201224014010-6772e930b67b // indirect
-	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5 // indirect
-	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a // indirect
-	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
-	golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 // indirect
-	google.golang.org/appengine v1.6.7 // indirect
-	gopkg.in/src-d/go-git.v4 v4.13.1
+	go.uber.org/zap v1.19.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.21.2
-	k8s.io/apiextensions-apiserver v0.20.1
-	k8s.io/apimachinery v0.21.2
-	k8s.io/client-go v12.0.0+incompatible
-	knative.dev/eventing v0.18.0
-	knative.dev/pkg v0.0.0-20200922164940-4bf40ad82aab
-	sigs.k8s.io/controller-runtime v0.6.3
+	k8s.io/api v0.20.7
+	k8s.io/apiextensions-apiserver v0.20.7
+	k8s.io/apimachinery v0.20.7
+	k8s.io/client-go v0.20.7
+	knative.dev/eventing v0.25.0
+	knative.dev/pkg v0.0.0-20210825070025-a70bb26767b8
+	sigs.k8s.io/controller-runtime v0.8.3
 	software.sslmate.com/src/go-pkcs12 v0.0.0-20210415151418-c5206de65a78
 )
 
 // local modules
 replace (
+	github.com/RHsyseng/operator-utils => github.com/vaibhavjainwiz/operator-utils v1.4.6-0.20210903063828-d5234ed62c72
```

Comment:
They didn't approve your PR? I think we should hold until they settle your updates.

Reply:
I had added `OnHold` label on this PR.

Reply:
@ricardozanini operator-utils PR has been merged now. I am removing `OnHold` label from this PR.